plugins/inputs/tacacs/sample.conf

```diff
@@ -0,0 +1,15 @@
+# Tacacs plugin collects successful tacacs authentication response times.
+[[inputs.tacacs]]
+  ## An array of Server IPs (or hostnames) and ports to gather from. If none specified, defaults to localhost.
+  # servers = ["127.0.0.1:49"]
+
+  ## Request source server IP, normally the server running telegraf.
+  # request_ip = "127.0.0.1"
+
+  ## Credentials for tacacs authentication.
+  username = "myuser"
+  password = "mypassword"
+  secret = "mysecret"
+
+  ## Maximum time to receive response.
+  # response_timeout = "5s"
```

Comment:
Please add a newline at the end of the file.

Reply:
added newline, added condition for timeout